src/styles/header.scss

```diff
@@ -0,0 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: white;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, Arial, sans-serif;
+  text-transform: uppercase;
+}
```

Comment:
You have a lot of properties in your `.header` class. It could be beneficial to group these properties by their function. For example, you could group together all the flex properties, all the font properties, etc. This makes your code more readable and maintainable.